src/color.js

```diff
@@ -182,6 +182,12 @@ export default class Color {
 			}
 		}
 	}
+
+	static isColorInstance = util.isInstance(this);
+
+	static isColor (arg) {
+		return Color.isColorInstance(arg) || Array.isArray(arg?.coords);
```

Comment:
I think the second part of the check should test to see if `arg` conforms to the `PlainColorObject` interface.